crates/matrix-sdk/src/room/mod.rs

```diff
@@ -2270,6 +2270,19 @@ impl Room {
         self.send_state_event(RoomTopicEventContent::new(topic.into())).await
     }
 
+    /// Removes the topic from this room if one is set.
+    pub async fn remove_room_topic(&self) -> Result<()> {
+        let event = self
```

Comment:
Could you add a smoke test to check that this calls the correct requests? The `MatrixMockServer` should make this relatively easy.

Let me know if you need some guidance on how the `MatrixMockServer` is supposed to be used.

Reply:
Ah nice. Just waiting to hear back if we actually want this method now given https://github.com/element-hq/element-meta/issues/2705#issuecomment-2618827011. Will come back and add that test if we do 👍